### DISCUSSION TITLE
Select Signals based on a Key

### DISCUSSION THREAD

**Discussion by danielmcn** [datalust/seq-tickets#1519] [General]
Hi

Not sure if this something that exists (help) or something that would need added (idea), but I can't find this scenario in the docs or on here.

We have Seq deployed in 3 instances company-wide - Dev, QA, and Prod, however creating ~~filters~~ signals in each environment results in them having different Ids, making it hard to compare them by just changing the URL.  Is there, or could we add, a way to create and select signals based on a defined key rather than Id?

e.g. we currently have to do
https://seq-dev:5341/#/events?signal=signal-103,signal-24
https://seq-qa:5341/#/events?signal=signal-82,signal-21
https://seq-prod:5341/#/events?signal=signal-64,signal-19

vs something like
 https://seq-dev:5341/#/events?signal=signal-ReportingApplications,signal-ServerPool2
   https://seq-qa:5341/#/events?signal=signal-ReportingApplications,signal-ServerPool2
https://seq-prod:5341/#/events?signal=signal-ReportingApplications,signal-ServerPool2

**Comment by nblumhardt** [comment]
Hi Daniel,

No, unfortunately - there's nothing I can think of that would support this right now. Setting up matching dashboards on each instance might be a reasonable approach in some cases.

Interesting thought, though, thanks for the note!

Best regards,
Nick
